allauth/socialaccount/providers/metamask/templates/metamask/auth.html

```diff
@@ -0,0 +1,73 @@
+{% load static %}
+<div id="personalSign">  </div>
+<HR>
```

Comment:
Is there a particular reason you used `<HR>` and not `<hr>`?

Reply:
I'm dropping these divs and HRs, they were for debug mostly.

Reply:
Latest PR to my local branch removes debugs